skycoin/skywire
# Make app-status generic

Did you run `make format && make check`?
yes

Fixes #1182 

 Changes:	
- Changed app-status from VPN specific to generic

How to test this PR:
1. Start visor `./skywire-cisor -c skywire-config.json`
2. Open UI start and stop all apps and see status.
